Cargo.toml

```diff
@@ -36,6 +36,7 @@ log = { version = "0.4", default-features = false }
 qlog = { version = "0.15.1", default-features = false }
 quinn-udp = { version = "0.5.11", default-features = false, features = ["direct-log"] }
 regex = { version = "1.9", default-features = false }
+rustc-hash = { version = "1.1", default-features = false, features = [ "std" ]}
```

Comment:
Firefox seems to use `rustc-hash` `2.1.1`:

https://searchfox.org/mozilla-central/rev/6a72a6d20eeb1b20b93862a79166938d6ce794a0/Cargo.lock#5846-5850